### PR TITLE
rfe #1441 Add regexp match when using AllowArbitraryServer

### PR DESCRIPTION
**Pull request by madhuracj**
Signed-off-by: Madhura Jayaratne madhura.cj@gmail.com

### CODE REVIEW

File: libraries/config/messages.inc.php [hunk 1]
```diff
@@ -17,6 +17,12 @@
 $strConfigAllowArbitraryServer_desc
     = __('If enabled, user can enter any MySQL server in login form for cookie auth.');
 $strConfigAllowArbitraryServer_name = __('Allow login to any MySQL server');
+$strConfigArbitraryServerRegexp_desc = __(
+    'Restricts the MySQL servers the user can enter when login to an arbitrary '
+    . 'MySQL server is anabled by matching the IP of the MySQL server to the given '
```
Comment (D-storm) [inline libraries/config/messages.inc.php]: Typo: anabled

File: libraries/config/ServerConfigChecks.class.php [hunk 2]
```diff
@@ -368,11 +368,14 @@ protected function performConfigChecksCookieAuthUsed(
      */
     protected static function defineMessages()
     {
-        $sAllowArbitraryServerWarn = __('This %soption%s should be disabled as it allows attackers to bruteforce login to any MySQL server. If you feel this is necessary, use %strusted proxies list%s. However, IP-based protection may not be reliable if your IP belongs to an ISP where thousands of users, including you, are connected to.');
+        $sAllowArbitraryServerWarn = __('This %soption%s should be disabled as it allows attackers to bruteforce login to any MySQL server. If you feel this is necessary, use %srestrict login to MySQL server%s or %strusted proxies list%s. However, IP-based protection with strusted proxies list may not be reliable if your IP belongs to an ISP where thousands of users, including you, are connected to.');
```
Comment (D-storm) [inline libraries/config/ServerConfigChecks.class.php]: Typo: strusted
Location:  IP-based protection with strusted proxies list

File: libraries/config/messages.inc.php (hunk 1 above)
Comment (madhuracj) [inline libraries/config/messages.inc.php]: Thanks!

File: libraries/config/ServerConfigChecks.class.php (hunk 2 above)
Comment (madhuracj) [inline libraries/config/ServerConfigChecks.class.php]: Proceeding s is part of %s for sprintf

Comment (madhuracj) [inline libraries/config/ServerConfigChecks.class.php]: Oh I just saw the second instance

Comment (D-storm) [inline libraries/config/ServerConfigChecks.class.php]: There is another instance of "strusted" without %s

**Comment by lem9** [comment]
Thanks.

**Pull request merged**